prime-angular-frontend/src/app/shared/components/address-view/address-view.component.html

```diff
@@ -14,7 +14,7 @@
 
   <ng-container *ngTemplateOutlet="(hasAddress) ? addressIf : addressElse"></ng-container>
 
-  <button *ngIf="showRedirect"
+  <button *ngIf="showEditRedirect"
```

Comment:
Wait, I'm confused about this change and why it's necessary.  Might need to chat about it

Reply:
In there, it just links the new variable to the button.